appengine-managed-runtime/pom.xml

```diff
@@ -31,6 +31,7 @@
   <properties>
     <maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
     <appengine.api.version>1.9.40</appengine.api.version>
```

Comment:
This is probably no longer used, but if it is, it should be at least 1.9.48 if not .49